src/layouts/components/Homepage/InfopicBody.tsx

```diff
@@ -46,7 +46,7 @@ export const InfopicBody = ({
 
   return (
     <Editable.Section>
-      <FormControl isInvalid={!!errors.subtitle}>
+      <FormControl isRequired isInvalid={!!errors.subtitle}>
```

Comment:
> notifications and infobar had optional tags, which should be removed per design

@seaerchin so this will mandate people to key in some value? What happens to existing sites where ppl didnt fill them in?

Reply:
nope, this is entirely UI only HAHAHA